web/src/main/java/org/wso2/testgrid/web/api/TestPlanService.java

```diff
@@ -166,7 +167,8 @@ public Response getLogContent(@PathParam("id") String id, @QueryParam("truncate"
             String testGridArtifactLocation = TestGridUtil.getTestRunArtifactsDirectory(testPlan).toString();
             String bucketKey = Paths
                     .get(AWS_BUCKET_ARTIFACT_DIR, testGridArtifactLocation, Constants.TEST_LOG_FILE_NAME).toString();
-            ArtifactReadable artifactDownloadable = new AWSArtifactReader(AWS_REGION_NAME, AWS_BUCKET_NAME);
+            ArtifactReadable artifactDownloadable = new AWSArtifactReader(ConfigurationContext.
+                    getProperty(ConfigurationProperties.AWS_REGION_NAME), AWS_BUCKET_NAME);
```

Comment:
This is fine for now since all our deployments happen in one region. But in future, different builds may run in different regions. In that case, this is going to be a per-testplan parameter. Specifically, this will be configured as an input-parameter under the infrastructure provisioner config in the TestPlan.